test/Toggle/IsCurrentUserToggleActiveTest.php

```diff
@@ -121,6 +136,65 @@ public function WhenToggleInvisibleAndNotEnabledByPolicyButReleaseDateIsTodayOrI
     }
 
     /**
+     * @test
+     */
+    public function GivenToggleInvisibleAndNotSetByPolicyAndReleaseDateIsInTheFutureAndUserIsAutoSubscribed_ThenInactive()
+    {
+        $this->setupChecker( false, false, false, false, false, false, false );
```

Comment:
I'm finding this setup method quite unreadable.

This is how I would create a similar but more readable arrange:

```php

public function setUp() {
  parent::setUp();
  $this->isAdrianCommenting = true;
  $this->numberOfDaysInMonth = 30;
  $this->toggleVisibility = 0;
}

public function givenFebruaryAndVisibleToggle_ThenIsSpooky()
{
  $this->numberOfDaysInMonth = 28;
  $this->toggleVisibility = 100;
  $this->act();
  $this->assertSpooky();
}
```